client/src/modules/cohort/components/Cohorts.jsx

```diff
@@ -86,6 +87,16 @@ class Cohorts extends Component {
     }
   };
 
+  renderBreadcrumbs = () => {
+    const breadcrumbs = [{ name: 'cohorts', path: '/cohorts' }];
```

Comment:
those strings should be taken from some consts in case something changes in future